roles/contiv_network/tasks/main.yml

```diff
@@ -1,10 +1,25 @@
 ---
 # This role contains tasks for configuring and starting netmaster and netplugin service
 
+- name: remove older netmaster
```

Comment:
This can impact ansible runtimes if same playbook needs to be played twice.

We can instead during create steps check if we are at a desired version or not and take appropriate action.